exec/vector/src/main/java/org/apache/drill/exec/vector/AllocationHelper.java

```diff
@@ -26,14 +26,16 @@ public static void allocate(ValueVector v, int valueCount, int bytesPerValue) {
     allocate(v, valueCount, bytesPerValue, 5);
   }
 
-  public static void allocatePrecomputedChildCount(ValueVector v, int valueCount, int bytesPerValue, int childValCount){
-    if(v instanceof FixedWidthVector) {
+  public static void allocatePrecomputedChildCount(ValueVector v, int valueCount, int bytesPerValue, int childValCount) {
```

Comment:
Discouraged: single letter parameter/field names. Even **"vv"** would be easier to search, if needed.